src/browser/api_protocol/api_handlers/authorization.js

```diff
@@ -15,7 +15,7 @@ let _ = require('underscore');
 let log = require('../../log');
 let socketServer = require('../../transports/socket_server').server;
 let ProcessTracker = require('../../process_tracker.js');
-
+const rvmMessageBus = require('../../rvm/rvm_message_bus').rvmMessageBus;
```

Comment:
meh, this is still js, if we want to get off Babel one day pre TS conversion we will  just have to switch it back